ififuncs.py

```diff
@@ -1510,4 +1510,41 @@ def get_ffprobe_dict(source):
     cmd = ['ffprobe', '-v', '0', '-show_versions', '-show_streams', '-show_format', '-print_format', 'json', source]
     ffprobe_json = subprocess.check_output(cmd)
     ffprobe_dict = json.loads(ffprobe_json)
+    print ffprobe_json
     return ffprobe_dict
+
+def get_colour_metadata(ffprobe_dict):
+    '''
+    Parses through a ffprobe dictionary and extracts colour metadata.
+    FFmpeg options are returned. Basically, the source metadata will be respected,
```

Comment:
I should change this part of the docstring as this function is not going to assume or stipulate anything - it will just generate some ffmpeg options based on the values in the source file. Any assumptions can be left for another function.